content/departments/cloud/technical-docs/v2.0/creation_process.md

```diff
@@ -17,59 +17,65 @@ cd deploy-sourcegraph-cloud-dev
 > See flow chart https://app.excalidraw.com/s/4Dr1S6qmmY7/2wUSD4kIxRo
 
 ```sh
```

Comment:
@michaellzc I would propose that we changed it into numbered list to make it more clear, as we have semi-automate process i.e.

1. Set environment variables
...
2. Create GCP project
`mgv2 ..`
`cd xxx && terraform apply`
3. Create infrastructure
`mgv2 ..`
`cd xxx && terraform apply`
4. Deploy application via kubectl/Helm
....

WDYT?

Reply:
The disadvantage of lists is that you can't link to specific items - we can add the numbers to the headings, but then links break as the ordering changes. IMO a best of both worlds approach is to have a numbered TOC with generic sections:

```
1. [Set environment variables](#set-environment-variables)
2. [Create infrastructure(#create-infrastructure)
3. ...
```